pytorch_lightning/trainer/trainer.py

```diff
@@ -124,7 +129,7 @@ def __init__(
 
                     Trainer(logger=logger)
 
-            checkpoint_callback (:class:`CheckpointCallback`): Callback for checkpointing.
+            checkpoint_callback: Callback for checkpointing.
```

Comment:
why remove the link to the class here? 
these should be left how they were

Reply:
these get generated automatically now because of the type annotations. It still shows in the docs even though it is removed from the docstring. Also, it generates a clickable link (before it was not clickable).

**Before:** 
![before](https://user-images.githubusercontent.com/5495193/75084098-105afe00-551e-11ea-83ba-8639e546e44c.png)

**After:**
![after](https://user-images.githubusercontent.com/5495193/75084110-2cf73600-551e-11ea-916a-67efab5879ff.png)

If this is not what you want, I can revert the changes. However, it would mean that in case an argument changes, the developer has to change the type in two places. It could lead to inconsistency.